canonical/snapcraft.io
# feat: add iot contact form

## Done
- Added contact-us page, and contact form modal
- Added links to contact form on:
  - Footer
  - /iot
  - Any snap details with the iot category

## How to QA
- Visit https://snapcraft-io-4542.demos.haus/about/contact-us
- Visit any page and click Contact Us in the footer
- Visit https://snapcraft-io-4542.demos.haus/iot and click "Contact us" under the main intro text
- Visit https://snapcraft-io-4542.demos.haus/node-red and see the contact link in the header under the install button - this is for any snap with the devices and iot category.
- filling in the form and clicking submit should redirect to https://snapcraft.io/about/thank-you, but won't work as that pages doesn't exist on snapcraft.io yet...

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-7541
